### ISSUE TITLE
[Bug]: Toolbar - Elements with aria-hidden="true" must not contain focusable elements.

### ISSUE THREAD

**Issue by katiearriagam** [microsoft/fluentui#27056]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Xeon(R) W-2133 CPU @ 3.60GHz
    Memory: 5.48 GB / 31.66 GB
  Browsers:
    Chrome: 110.0.5481.178
    Edge: Spartan (44.22621.1265.0), Chromium (110.0.1587.63)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/s/t15v6m?file=/example.tsx

## Repro steps
1. Open Code Sanbox: https://codesandbox.io/s/t15v6m?file=/example.tsx
2. Open the example in a new window 
![Screenshot of the example showing how to open in a new window](https://user-images.githubusercontent.com/14140121/222868403-88b3b346-cd43-46d4-842d-c0c49ef547fc.png)
3. Run the Accessibility Insights for Web by Microsoft: 
![screenshot of results of the accessibility insights for web tool](https://user-images.githubusercontent.com/14140121/222868444-910961ea-fc2a-4054-a375-707698d4180d.png)

## Actual Behavior
> Title: WCAG 4.1.2: Ensures aria-hidden elements are not focusable nor contain focusable elements (i:nth-child(5))
Tags: Accessibility, WCAG 4.1.2, aria-hidden-focus

> Issue: Ensures aria-hidden elements are not focusable nor contain focusable elements (aria-hidden-focus - https://accessibilityinsights.io/info-examples/web/aria-hidden-focus)

This happens because the `data-tabster-dummy` has `aria-hidden="true"` and `tabindex="0"`

## Expected Behavior
Either set tabindex to -1 or aria-hidden should not be set to true.

### Requested priority

High

### Products/sites affected

Office Online suite

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by akurtaliqi** [comment]
Hi, I would like to pick this issue. I'm a beginner as contributor in this project. Do you think it's doable ?
